client/app/src/views/userScreenshots.html

```diff
@@ -1 +1,11 @@
-Hello from userScreenshots.html
+<!-- Template for UserScreenshots -->
```

Comment:
High-level question: will userScreenshots.html always be the same as the landing page screenshots template? If so, we could have the view point to the same template.